dags/kube_secrets.py

```diff
@@ -62,3 +62,4 @@
 CLEARBIT_KEY = Secret("env", "CLEARBIT_KEY", "airflow", "CLEARBIT_KEY")
 
 GITHUB_TOKEN = Secret("env", "GITHUB_TOKEN", "airflow", "GITHUB_TOKEN")
```

Comment:
can we remove the existing GITHUB_TOKEN and the task using it ?

Reply:
Let's do cleanup as soon as we make sure this one is working.